l10n_it_fiscalcode/__manifest__.py

```diff
@@ -1,30 +1,23 @@
-# Copyright 2014 Associazione Odoo Italia (<http://www.odoo-italia.org>)
-# Copyright 2016 Andrea Gallina (Apulia Software)
-# Copyright 2018 Matteo Bilotta (Link IT s.r.l.)
-# Copyright 2018 Lorenzo Battistini (https://github.com/eLBati)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
-{
-    'name': 'Italian Localization - Fiscal Code',
-    'version': '12.0.1.0.0',
-    'development_status': 'Production/Stable',
-    'category': 'Localisation/Italy',
-    'author': "Link IT s.r.l., "
-              "Apulia Software, "
-              "Odoo Italia Network, "
-              "Odoo Community Association (OCA)",
-    'website': 'https://github.com/OCA/l10n-italy',
-    'license': 'AGPL-3',
-    'depends': ['base_vat'],
-    'external_dependencies': {
-        'python': ['codicefiscale'],
-    },
-    'data': [
-        "security/ir.model.access.csv",
-        'data/res.city.it.code.csv',
-        'view/fiscalcode_view.xml',
-        'view/report_invoice_document.xml',
-        'wizard/compute_fc_view.xml',
-        ],
-    'installable': True
-}
+# -*- coding: utf-8 -*-
+# Copyright 2014 Associazione Odoo Italia (<http://www.odoo-italia.org>)
+# Copyright 2016 Andrea Gallina (Apulia Software)
+# Copyright 2018 Matteo Bilotta (Link IT s.r.l.)
+# Copyright 2018 Lorenzo Battistini (https://github.com/eLBati)
+# Copyright 2018 Teuron (<http://www.teuron.it>)
+# Copyright 2018 RDS S.p.A. (<http://www.rdsmoulding.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Italian Localisation - Fiscal Code',
+    'version': '12.0.0.0.0',

```

Comment:
Version should probably be `12.0.2.0.0`.
See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers